src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+
+  let totalRent = 0;
+
+  if (days >= longTerm) {
+    totalRent = dayRent * days - longTermDiscount;
```

Comment:
You can remove the `else` keyword and simply use `if` with a `return` statement to simplify the code. Since you're returning within the `if` block, the subsequent code won't be executed if the condition is met.